web/packages/teleterm/src/services/tshd/interceptors.ts

```diff
@@ -90,3 +126,18 @@ export function filterSensitiveProperties(toFilter: object): object {
 
   return acc;
 }
+
+function makeMethodLogger(logger: Logger, methodName: string) {
+  return {
+    logRequest: (input: object) => {
+      logger.info(`${methodName} REQUEST:`, filterSensitiveProperties(input));
+    },
+    logResponse: (output: object) => {
+      const toLog = output ? filterSensitiveProperties(output) : null;
+      logger.info(`${methodName} RESPONSE:`, toLog);
+    },
+    logError: (error: unknown) => {
+      logger.error(`${methodName} RESPONSE:`, `${error}`);
```

Comment:
I feel like logs are a little less noisy when it's not uppercased:

```suggestion
      logger.info(`${methodName} request:`, filterSensitiveProperties(input));
    },
    logResponse: (output: object) => {
      const toLog = output ? filterSensitiveProperties(output) : null;
      logger.info(`${methodName} response:`, toLog);
    },
    logError: (error: unknown) => {
      logger.error(`${methodName} response:`, `${error}`);
```